src/sql/workbench/contrib/notebook/browser/notebookActions.ts

```diff
@@ -308,6 +309,16 @@ export class RunParametersAction extends TooltipFromLabelAction {
 	*/
 	public async run(context: URI): Promise<void> {
 		const editor = this._notebookService.findNotebookEditor(context);
+		// Only run action for kernels that are supported (Python, PySpark, PowerShell)
+		let kernelsNotSupported = ['sql', 'kusto'];
```

Comment:
These kernel values should be put as a const somewhere - didn't we have you do that for something else a little while back? If you haven't then I would suggest adding that - adding a string enum of kernels seems like a very useful thing to have.